.chloggen/msg_fix-docker-observer.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: dockerobserver 
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Including docker observer as part of the contrib release.
+
+# One or more tracking issues related to the change
+issues: [21521]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: -|
+  Fixes issue [collector-release#314](https://github.com/open-telemetry/opentelemetry-collector-releases/issues/314)
```

Comment:
I'd avoid adding a log entry here

Reply:
Is it just worth adding to the main text?

Reply:
I mean I don't think we need a log entry here at all since this just ads the component to the developer build

Reply:
I had missed the change that we are dogfooding the collector builder, was meant for the actual distro releases.

Reply:
So can we remove this file (I can add `skip changelog` label) and remove `Closes https://github.com/open-telemetry/opentelemetry-collector-releases/issues/314` because it doesn't close it?

Reply:
I'll open a PR there also :)